test/linting/lint.sh

```diff
@@ -105,7 +104,7 @@ function _shellcheck
     -not -path './target/docker-configomat/*'
   )"
   # macOS lacks parity for `-executable` but presently produces the same results: https://stackoverflow.com/a/4458361
-  [[ "$(uname)" == "Darwin" ]] && FIND_EXEC="-perm +111 -type l -or" || FIND_EXEC="-executable"
+  [[ "$(uname)" == "Darwin" ]] && FIND_EXEC="-perm -711" || FIND_EXEC="-executable"
```

Comment:
This change isn't mentioned in the PR description. Can you add a short description for future reference?